frontend/src/modules/organization/components/list/organization-list-table.vue

```diff
@@ -838,6 +838,13 @@ const onTableMouseLeft = () => {
   isScrollbarVisible.value = isCursorDown.value;
 };
 
+const truncateText = (text, characters = 200, suffix = '') => {
+  if (text.length > characters) {
+    return `${text.substring(0, characters)}` + suffix;
+  }
+  return text;
+}
+
```

Comment:
Since this is a util that could be reused in other situations could you move it to `utils/string.js`?

Reply:
Done